instance_skel_types.d.ts

```diff
@@ -111,6 +111,7 @@ export interface CompanionInputField {
 	type: 'text' | 'textinput' | 'textwithvariables' | 'dropdown' | 'colorpicker' | 'number' | 'checkbox'
 	label: string
 	tooltip?: string
+	isVisible?: () => boolean
```

Comment:
This should be `isVisible?: (config: { [id: string]: InputValue }) => boolean`

Reply:
You're right. I added it as a placeholder and forgot to go back and update. Will fix.